packages/react/src/components/Carousel/index.tsx

```diff
@@ -57,12 +58,12 @@ const FULL_SCREEN_WIDTH = 100
 const USER_DRAG_LENGTH = 100
 
 export const Carousel = forwardRef(function Carousel(
-  { images, isArrow, size = 687, name, ...props }: CarouselProps,
+  { images = [], isArrow, size = 687, name, ...props }: CarouselProps,
```

Comment:
```suggestion
  { images, isArrow, size = 687, name, ...props }: CarouselProps,
```

아래에서 옵셔널 체이닝으로 방어 코드 작성되어 있어서 제거해도 괜찮을 것 같습니다!

Reply:
[완료](https://github.com/price-offer/offer-design-system/pull/253/commits/5c5b984e5c7c51d0edc2defa34ee8c2be9f88b04#diff-872148be87266ca1e7c5655807039375fc9a31a44fde82233e8244482b67ffc4R66)